code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -1762,4 +1836,4 @@
 
 /obj/item/weapon/storage/backpack/fluff/stunstaff/New()
 	..()
-	new /obj/item/weapon/melee/baton/fluff/stunstaff(src)
```

Comment:
What does this have to do with the egg implant?

Reply:
I have no idea. I didn't touch those lines, github did.

Reply:
Ah it was just a line ending change.